infracheck/checks/domain-expiration

```diff
@@ -29,16 +31,23 @@ import re
 import time
 import subprocess
 from collections import namedtuple
+from whois._3_adjust import str_to_date
 
 
-ManualCheck = namedtuple('ManualCheck', 'expiration_date')
+WhoisResult = namedtuple('ManualCheck', 'expiration_date')
 
 
-class DomainCheck:
+class DomainCheck(object):
```

Comment:
What's the difference between no parameter and object parameter?

Reply:
Today - no reason, just looks better :joy: 
Historicaly: https://stackoverflow.com/a/45062077/6782994